test/functional/change_stream_tests.js

```diff
@@ -1876,6 +1877,45 @@ describe('Change Streams', function() {
       .then(() => teardown(), teardown);
   });
 
+  it('should emit close event after error event', {
+    metadata: { requires: { topology: 'replicaset', mongodb: '>=3.5.10' } },
+    test: function(done) {
+      const configuration = this.configuration;
+      const client = configuration.newClient();
+      const closeSpy = sinon.spy();
+
+      client.connect(function(err, client) {
+        expect(err).to.not.exist;
+
+        const db = client.db('integration_tests');
+        const coll = db.collection('event_test');
+        const changeStream = coll.watch([{ $project: { _id: false } }]);
```

Comment:
maybe leave a comment here indicating why this pipeline will generate an error, I spent a minute like "is an error even triggered here :thinking:"